pkg/cache/queue_info_test.go

```
@@ -56,10 +59,7 @@ func createUnManagedQueue(parentQI *QueueInfo, name string, parent bool) (*Queue
 // base test for creating a managed queue
 func TestQueueInfo(t *testing.T) {
 	// create the root
-	root, err := createRootQueue()
-	if err != nil {
-		t.Fatalf("failed to create basic root queue: %v", err)
-	}
+	root := createRootQueue(t)
```

This will hide the error from the caller and make tracking more difficult